src/main/java/com/microsoft/aad/msal4j/ITenantProfile.java

```diff
@@ -3,14 +3,15 @@
 
 package com.microsoft.aad.msal4j;
 
+import java.io.Serializable;
 import java.util.Map;
 
 /**
  * Interface representing a single tenant profile. ITenantProfiles are made available through the
  * {@link IAccount#getTenantProfiles()} method of an Account
  *
  */
-public interface ITenantProfile {
+public interface ITenantProfile extends Serializable {
```

Comment:
I have noticed that ? is used as value of claim, we probably should switch to String

Reply:
The JWT parser we use (from the Nimbus package) returns the claims as a `Map<String, Object>` , and I don't think there's an easy way to convert that to a  `Map<String, String>`  (other than going through each item and getting the String value). 

Because of that, I could make claims either a `Map<String, Object>` or a `Map<String, ?>` . A `Map<String, Object>` can only passed into methods and whatnot that explicitly have a parameter of `Map<String, Object>`, whereas I believe `Map<String, ?>` could be passed as a parameter of type `Map<String, String>` (or another String/object pair), which I figured would be more useful for users.